web/docs/project/SecretGeneratorBlock.module.css

```diff
@@ -0,0 +1,51 @@
+@keyframes popUp {
+  from {
+    transform: translateY(6px);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+.generateBtn {
+  display: inline-block;
+  padding: 6px 12px;
+  font-size: 0.9rem;
+  line-height: 1rem;
+  border: 1px solid var(--ifm-color-emphasis-200);
```

Comment:
Can you try out having a less muted color here? It doesn't have to be super bright, but I almost can't see the border with such a light grey 😔